openedx/adg/lms/applications/admin.py

```diff
@@ -396,7 +397,7 @@ def _send_application_status_update_email(self, application, message_for_applica
             'message_for_applicant': message_for_applicant,
         }
 
-        MandrillClient().send_mandrill_email(
+        task_send_mandrill_email.delay(
```

Comment:
I think this function will break when it will run celery environment, you need to pass serializable object to celery task